tests/promo-code/promo-code.mutation.test.js

```diff
@@ -1,32 +1,49 @@
-const { newPromoCode } = require('./promo-code.variables');
+const {
+  newPromoCode,
+  newPromoCodeForUpdate,
+} = require('./promo-code.variables');
 const { setupApp } = require('../helper-functions');
-
-const { addPromoCode, deletePromoCode } = require('./promo-code.helper');
+const {
+  addPromoCode,
+  deletePromoCode,
+  updatePromoCode,
+} = require('./promo-code.helper');
 
 let promoCodes;
 let promoCodeId;
 let operations;
 
-describe('PromoCodes queries', () => {
-  beforeAll(async () => {
+describe('PromoCodes mutations', () => {
+  beforeEach(async () => {
     operations = await setupApp();
+    promoCodes = await addPromoCode(newPromoCode, operations);
```

Comment:
Not sure that we need add promocode before each test, same as call setupApp function. I think it' more proper to use beforeAll here.

Reply:
I agree. Improved this